<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogVecDisplay</class>
 <widget class="QDialog" name="dialogVecDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Vector Overlay Files</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file(s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Default color (if not specified in file)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Red [0,1]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Green [0,1]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue [0,1]</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude offset (m)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Overwrite all map objects so always visible</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shapefile parameters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="redEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="grnEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="bluEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="altEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="withButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Render with map</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="overButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Overwrite map (filled polygons show artifacts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Polygon fill type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude modes</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="wireButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wireframe</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="filledButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Filled </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="clampButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Clamp to ground</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="relButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Relative to ground</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="absoluteButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Absolute</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Relative to attribute GrndHae</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>481</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wireButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doWire()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filledButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doFilled()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clampButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doClamp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>relButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doRel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>absoluteButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doAbsolute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doAttr()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>withButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doWith()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>overButton</sender>
   <signal>clicked()</signal>
   <receiver>dialogVecDisplay</receiver>
   <slot>doOver()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doBrowse()</slot>
  <slot>doLoad()</slot>
  <slot>doWire()</slot>
  <slot>doFilled()</slot>
  <slot>doClamp()</slot>
  <slot>doRel()</slot>
  <slot>doAbsolute()</slot>
  <slot>doAttr()</slot>
  <slot>doWith()</slot>
  <slot>doOver()</slot>
 </slots>
</ui>
